### PR TITLE
Replace PNGs by SVGs

### PR DESCRIPTION
**Pull request by phallobst**

### CODE REVIEW

File: img/c/commuterbenefits.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 132.5 132.8"><g transform="translate(-110 -45.5)"><circle cx="176" cy="111.7" r="66.1" fill="#fff" fill-rule="evenodd"/><path fill="#ee2e24" d="M176 178.4h-.2a66.7 66.7 0 0 1-55.2-30.7l.3.3c4.2 5.5 11 6.2 11 6.2 1.6.3 3.4.5 5.3.5h38.9l2.9-17.5h-35c-5.4 0-8.3-3.2-8.2-9.8v-9.8H170l3-17h-37.1V81.1h41l2.9-17.6h-49.1c11.9-11 28-18 45.4-18h.4a66.3 66.3 0 0 1-.6 132.9"/></g></svg>
```
Comment (Carlgo11) [inline img/c/commuterbenefits.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 132.2 132.2"><circle cx="66" cy="66" r="66" fill="#ee2e24"/><path fill="#fff" d="M20.8 17.9a65.9 65.9 0 0 0-9.8 84.6c4 5.5 11 6.1 11 6.1 1.6.3 3.4.5 5.3.5h38.9l2.9-17.5h-35c-5.4 0-8.3-3.2-8.2-9.8V72h34.2l3-17H26V35.5h41l2.9-17.6z"/></svg>
```

File: entries/c/commuterbenefits.com.json [hunk 2]
```diff
@@ -1,12 +1,11 @@
 {
   "CommuterBenefits": {
     "domain": "commuterbenefits.com",
-    "img": "commuterbenefits.com.png",
     "contact": {
       "twitter": "CommuterBenefit"
```
Comment (Carlgo11) [inline entries/c/commuterbenefits.com.json]:
```suggestion
      "twitter": "EdenredBenefits"
```

File: entries/c/commuterbenefits.com.json [hunk 3]
```diff
@@ -1,12 +1,11 @@
 {
   "CommuterBenefits": {
```
Comment (Carlgo11) [inline entries/c/commuterbenefits.com.json]: Based on their lack of GDPR compliance I'd say they seem to only cater to the U.S. market.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/c/commuterbenefits.com.json (hunk 3 above)
Comment (phallobst) [reply entries/c/commuterbenefits.com.json]: "Edenred" is international, but "Commuter Benefits" seems to be US only.

**Comment by fletcherdodd** [comment]
Is the Epic Games one an accident? I just changed that one😟

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**